### PR TITLE
Migrations

### PR DESCRIPTION
**Pull request by karbmk**
Migration for Event private and public. UserTimeLog duration field added

### CODE REVIEW

File: openCurrents/migrations/0030_auto_20170725_1500.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+    dependencies = [
+        ('openCurrents', '0029_auto_20170724_1517'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='event',
+            name='is_public',
+            field=models.BooleanField(default=True),
```
Comment (nickolashe) [inline openCurrents/migrations/0030_auto_20170725_1500.py]: @karbmk actually, lets use the default value of `False` as in your initial change. i don't think we want to create any currents without explicit approval.

File: openCurrents/models.py [hunk 2]
```diff
@@ -157,6 +157,8 @@ class Event(models.Model):
         default=GROUP
     )
 
+    is_public = models.BooleanField(default=True)
```
Comment (nickolashe) [inline openCurrents/models.py]: @karbmk is this field used anywhere yet?

File: openCurrents/models.py [hunk 3]
```diff
@@ -218,6 +220,8 @@ class UserTimeLog(models.Model):
     datetime_start = models.DateTimeField('start time')
     datetime_end = models.DateTimeField('end time', null=True, blank=True)
 
+    datetime_duration = models.CharField(max_length=20, default='00:00:00')
```
Comment (nickolashe) [inline openCurrents/models.py]: @karbmk lets make this a decimal field (measured in hours)

Comment (karbmk) [inline openCurrents/models.py]: Ok got it

File: openCurrents/models.py (hunk 2 above)
Comment (karbmk) [inline openCurrents/models.py]: Not yet

File: openCurrents/migrations/0029_auto_20170724_1517.py [hunk 4]
```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```
Comment (nickolashe) [inline openCurrents/migrations/0029_auto_20170724_1517.py]: I think we can delete this migration

**Pull request merged**